src/markdown-view.android.ts

```diff
@@ -1,29 +1,36 @@
-import { markdownProperty, MarkdownViewBase } from './markdown-view.common';
-
+import { fontSizeProperty, markdownProperty, MarkdownViewBase } from './markdown-view.common';
+​
 declare var ru: any;
-
+​
 export class MarkdownView extends MarkdownViewBase {
-
+​
     _android: any = null; // android.widget.TextView
-
+​
     markwon: any;
-
+​
+    private _fontSize = 14;
+​
     constructor() {
         super();
     }
-
+​
     get android(): android.widget.TextView {
         return this._android;
     }
-
+​
     public createNativeView() {
         this.markwon = ru.noties.markwon.Markwon.create(this._context);
         this._android = new android.widget.TextView(this._context);
         return this._android;
     }
-
+​
+    [fontSizeProperty.setNative](size: number) {
+        this._fontSize = size;
+    }
+​
     [markdownProperty.setNative](markdown: string) {
+        this._android.setTextSize(Number(this._fontSize));
```

Comment:
This should probably be done when the `fontSize` property is set, else it will not update the font size when the property is updated